redirects.csv

```diff
@@ -47,7 +47,7 @@ https://discipleship.crossroads.net/*,https://www.crossroads.net/huddle,301!,mas
 /mycamps*,https://${env:CRDS_ANGULAR_ENDPOINT}/mycamps:splat,200!
 /camps/:campId/*,https://${env:CRDS_ANGULAR_ENDPOINT}/camps/:campId/,200!
 /childcare/*,https://${env:CRDS_CHILDCARE_ENDPOINT}/:splat,200!
-/corkboard*,https://${env:CRDS_ANGULAR_ENDPOINT}/corkboard:splat,200!
+/corkboard*,https://${env:CRDS_CORKBOARD_ENDPOINT}/:splat,200!
```

Comment:
This may change or we may need to get the correct value for `CRDS_CORKBOARD_ENDPOINT`. If you go to `/corkboard` on `int` and `prod` it works. On `demo` and this deploy preview, it doesn't load.